src/packages/contracts/utilities/getUniqueContractAddress/index.ts

```diff
@@ -7,5 +7,10 @@ export type GetUniqueContractAddressInput = {
   chainId: ChainId;
 };
 
-export const getUniqueContractAddress = ({ name, chainId }: GetUniqueContractAddressInput) =>
-  addresses[name][chainId];
+export const getUniqueContractAddress = ({ name, chainId }: GetUniqueContractAddressInput) => {
+  const contractAddresses = addresses[name];
+
+  return Object.prototype.hasOwnProperty.call(contractAddresses, chainId)
```

Comment:
`hasOwn` [can be used here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn)

```suggestion
  return Object.hasOwn(contractAddresses, chainId)
```

Reply:
Nice, I didn't know that existed! Thanks